tests/Unit/PointwiseFunctions/Hydro/EquationsOfState/Python/Test_Enthalpy.py

```diff
@@ -0,0 +1,156 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import json
+import sys
+import unittest
+
+"These imports will be needed for second merge request"
+# sys.path.append("..")
+# import Enthalpy as enthalpy_python
+# import Enthalpy.eos_evaluator as enthalpy_eos
+import numpy as np
+import numpy.testing as npt
+import pandas as pd
+import scipy.integrate as integrate
+import scipy.interpolate as interpolate
+
+import spectre
+import spectre.PointwiseFunctions.Hydro.EquationsOfState as EoS
+from spectre.DataStructures import DataVector
+from spectre.DataStructures.Tensor import Scalar
+
+# import Spectral as spectral_python
+
+
+class TestEnthalpy(unittest.TestCase):
+    def test_enthalpy(self):
+        ##Set parameters:
+
+        low_eos_reference_density = 4.533804669935759e-05
```

Comment:
just put these numbers directly into the constructors below, since you don't use them multiple times.

Also add a comment how you chose these numbers.